packages/ra-supabase-core/src/useSetPassword.ts

```diff
@@ -0,0 +1,42 @@
+import {
+    OnFailure,
+    OnSuccess,
+    useAuthProvider,
+    useNotify,
+    useRedirect,
+} from 'ra-core';
+import { SetPasswordParams, SupabaseAuthProvider } from './authProvider';
+
+/**
+ * This hook returns a function to call in order to set a user password on Supabase.
+ **/
```

Comment:
could you add basic usage example in the jsdoc?

Reply:
Done